lib/apiservers/engine/backends/backends.go

```diff
@@ -92,44 +93,89 @@ func Init(portLayerAddr, product string, config *config.VirtualContainerHostConf
 	// the vic-machine installer timeout will intervene if this blocks for too long
 	pingPortLayer()
 
+	if err := hydrateCaches(); err != nil {
+		return err
```

Comment:
All conditions should be fatal, and the first cache refresh that fails will be the one that is returned here (see line 174)